packages/gatsby/src/utils/websocket-manager.js

```diff
@@ -2,25 +2,49 @@ const path = require(`path`)
 const { store } = require(`../redux`)
 const fs = require(`fs`)
 
-const getCachedPageData = (pagePath, directory) => {
-  const { jsonDataPaths, pages } = store.getState()
-  const page = pages.find(p => p.path === pagePath)
-  const dataPath = jsonDataPaths[page.jsonName]
-  if (typeof dataPath === `undefined`) return undefined
+const readCachedResults = (dataPath, directory) => {
   const filePath = path.join(
     directory,
     `public`,
     `static`,
     `d`,
     `${dataPath}.json`
   )
-  const result = JSON.parse(fs.readFileSync(filePath, `utf-8`))
+  return JSON.parse(fs.readFileSync(filePath, `utf-8`))
+}
+
+const getCachedPageData = (pagePath, directory) => {
+  const { jsonDataPaths, pages } = store.getState()
+  const page = pages.find(p => p.path === pagePath)
+  const dataPath = jsonDataPaths[page.jsonName]
+  if (typeof dataPath === `undefined`) {
+    console.log(`Something wrong
```

Comment:
Maybe this could be:
```
Error loading a result for the StaticQuery in "${staticQueryComponent.componentPath}. Query was not run and no cached result was found.
```